dist/qs.js

```diff
@@ -308,6 +309,7 @@ var stringify = function stringify( // eslint-disable-line func-name-matching
     prefix,
     generateArrayPrefix,
     strictNullHandling,
+    skipBlanks,
```

Comment:
All changes made in this autogenerated file are lost; you'll need to change the files in `lib` instead.